syntax = "proto3";

package demo;

service DemoService {
  rpc Hello (HelloRequest) returns (HelloReply) {}
}

message HelloRequest {}
message HelloReply {}